client/extensions/woocommerce/app/order/index.js

```diff
@@ -54,14 +54,14 @@ class Order extends Component {
 	}
 
 	render() {
-		const { className, isSaving, order, site, translate } = this.props;
+		const { className, isSaving, order, orderId, site, translate } = this.props;
 		if ( ! order ) {
 			return null;
 		}
 
 		const breadcrumbs = [
 			( <a href={ getLink( '/store/orders/:site/', site ) }>{ translate( 'Orders' ) }</a> ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Order' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/16495-add-order-id)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).